packages/vscode-extension/package.nls.json

```diff
@@ -225,15 +225,21 @@
     "teamstoolkit.handlers.localStateFileNotFound": "State file not found in environment %s. You must run `debug` first to generate related state file.",
     "teamstoolkit.handlers.teamsAppIdNotFound": "Failed to find 'fx-resource-appstudio.teamsAppId' in 'state.%s.json'.",
     "teamstoolkit.handlers.defaultManifestTemplateNotExists": "Manifest template file not found in %s. You can use CLI with your own template file.",
+    "teamstoolkit.handlers.defaultAppPackageNotExists": "App package file not found in %s. You can use CLI with your own app package file.",
+    "teamstoolkit.handlers.validate.selectTitle": "Select a validation method",
+    "teamstoolkit.handlers.validate.schemaOption": "Validate using manifest schema",
+    "teamstoolkit.handlers.validate.schemaOptionDescription": "Validate using manifest schema",
+    "teamstoolkit.handlers.validate.appPackageOption": "Validate app package using validation rules",
+    "teamstoolkit.handlers.validate.appPackageOptionDescription": "Validate app package using validation rules",
     "teamstoolkit.localDebug.failedCheckers": "Failed to check: [%s]. ",
     "teamstoolkit.localDebug.installApp.bot.configureOutlook": "Configure Outlook",
-    "teamstoolkit.localDebug.installApp.bot.description": "To continue to debug your application in Outlook, you need to follow two steps:",
+    "teamstoolkit.localDebug.installApp.bot.description": "To continue debugging your application in Outlook, you need to follow two steps:",
```

Comment:
i think you don't need the "you need to follow two steps". So I would say:
"To continue debugging your application in Outlook:"